packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -105,7 +105,7 @@ function createLongTaskSpans(longtasks) {
 export function createFirstInputDelaySpan(fidEntries) {
   let firstInput = fidEntries[0]
 
-  if (firstInput && !metrics.wasHidden) {
+  if (firstInput) {
```

Comment:
Are we removing this check? since we are discarding the transaction anyway incase its hidden?

Reply:
the firstInput could still be undefined if the array is empty.

Reply:
I meant about the `wasHidden` check.